hydra-configs-torchvision/setup.py

```diff
@@ -9,5 +9,5 @@
     author_email=["omry@fb.com", "rosario@cs.uw.edu"],
     url="http://github.com/pytorch/hydra-torch",
     include_package_data=True,
-    install_requires=["omegaconf"],
+    install_requires=["omegaconf", "torchvision==0.7"],
```

Comment:
1. Why is this an install dependency?
2. Why are you limiting to a specific version? we can warn about mismatch it in a runtime check, right?

Reply:
I agree. This is not an install requirement. I'm going to move this to a `requirements/dev.txt` for this project since tests depend on it. 

For runtime, what module do you propose does this runtime check? Are you thinking we do this on import in `__init__.py`?

Reply:
Doing it on import seems reasonable, yes.

Reply:
done. It checks the configs version defined by `setup.py` and compares it to the currently installed `torchvision`.